protocol-designer/src/components/modals/FilePipettesModal/PipetteFields.tsx

```diff
@@ -106,30 +106,28 @@ export function PipetteFields(props: Props): JSX.Element {
     const filter96 = mount === RIGHT ? ['p1000_96'] : []
 
     return (
-      <Flex width="13.8rem">
-        <PipetteSelect
-          nameBlocklist={
-            robotType === OT2_ROBOT_TYPE
-              ? OT3_PIPETTES
-              : [...OT2_PIPETTES, ...filter96]
-          }
-          enableNoneOption
-          tabIndex={tabIndex}
-          pipetteName={pipetteName != null ? pipetteName : null}
-          onPipetteChange={pipetteName => {
-            const nameAccessor = `pipettesByMount.${mount}.pipetteName`
-            const value = pipetteName
-            const targetToClear = `pipettesByMount.${mount}.tiprackDefURI`
-            // this select does not return an event so we have to manually set the field val
-            onSetFieldValue(nameAccessor, value)
-            onSetFieldValue(targetToClear, null)
-            onSetFieldTouched(targetToClear, false)
-          }}
-          disabled={mount === RIGHT && values.left.pipetteName === 'p1000_96'}
-          id={`PipetteSelect_${mount}`}
-          className={styles.pipette_select}
-        />
-      </Flex>
+      <PipetteSelect
+        nameBlocklist={
+          robotType === OT2_ROBOT_TYPE
+            ? OT3_PIPETTES
+            : [...OT2_PIPETTES, ...filter96]
+        }
+        enableNoneOption
+        tabIndex={tabIndex}
+        pipetteName={pipetteName != null ? pipetteName : null}
+        onPipetteChange={pipetteName => {
+          const nameAccessor = `pipettesByMount.${mount}.pipetteName`
+          const value = pipetteName
```

Comment:
nit 
nameAccessorValue or accessorValue since value is a little bit vague 
```suggestion
          const nameAccessorValue = pipetteName
```